### PR TITLE
crgolf:cpp : Fix coinage DIPSW

### PR DESCRIPTION
**Pull request by sasuke-arcade**
- Applied version differences
- Distinguishing between 2Coin/1Credit and 2Coin/1Play

### CODE REVIEW

File: src/mame/drivers/crgolf.cpp [hunk 1]
```diff
@@ -458,6 +458,27 @@ static INPUT_PORTS_START( crgolf )
 INPUT_PORTS_END
 
 
+static INPUT_PORTS_START(crgolfa)
+	PORT_INCLUDE(crgolf)
+
+	PORT_MODIFY("DSW")
+	PORT_DIPNAME( 0x20, 0x20, DEF_STR( Coinage ) ) PORT_DIPLOCATION( "SW:5" )
+	PORT_DIPSETTING( 0x20, "2 Coins/1 Play" )
+	PORT_DIPSETTING( 0x00, "1 Coins/1 Play" )
```
Comment (cuavas) [inline src/mame/drivers/crgolf.cpp]: What's the reason for not using the standard `DEF_STR( 2C_1C )` and `DEF_STR( 1C_1C )` for this set?

Comment (sasuke-arcade) [reply src/mame/drivers/crgolf.cpp]: These sets are always 1 credit for 1 coin.
DIPSW will set credits (=coin) count for starting one play.

Comment (MooglyGuy) [reply src/mame/drivers/crgolf.cpp]: That's not the point, the point is that using `DEF_STR( 2C_1C )` and `DEF_STR( 1C_1C )` is localizable into different languages. Arbitrary strings are not.

Comment (sasuke-arcade) [reply src/mame/drivers/crgolf.cpp]: Is it correct to write "1 Coin/2 Credits" even in the case of "1 Coin/1 Credit, Start for 2 Credit"? If that is correct, I will change it.

Comment (cuavas) [reply src/mame/drivers/crgolf.cpp]: If it's the price to start, then you should be more explicit – as it is, it looks like using custom strings for credits per coin.  You should use "Price to start", "1 Credit", "2 Credits", or "Credits to start", "1", "2".  That would make what it does clearer.

Comment (sasuke-arcade) [reply src/mame/drivers/crgolf.cpp]: Thanks for your advice. Changed to "Credits To Start".

**Pull request merged**